### PR TITLE
CREATE_BLADERUNNER: Add support for python3

### PR DESCRIPTION
**Pull request by phcoder**
I tried to keep python2 compatibility but didn't check as python2 is deprecated and problematic at this point

### CODE REVIEW

File: devtools/create_bladerunner/subtitles/mixResourceCreator/packBladeRunnerMIXFromPCTLKXLS04.py [hunk 1]
```diff
@@ -252,14 +252,14 @@ def initOverrideEncoding(pathToConfigureFontsTranslationTxt):
 									and fontCateg_targetEnc_OOOGlyphs_Tuple[1] != ''):
 								tmpFontCateg = fontCateg_targetEnc_OOOGlyphs_Tuple[0]
 								tmpTargetEncodingForThisFont = fontCateg_targetEnc_OOOGlyphs_Tuple[1]
-								if ( tmpFontCateg not in zip(*DEFAULT_TARGET_ENCODING_PER_FONT)[0]):
+								if ( tmpFontCateg not in list(zip(*DEFAULT_TARGET_ENCODING_PER_FONT))[0]):
 									print ('[Error] Invalid Font name specified in configureFontsTranslation text file!')
 									print ('        Valid values are: ', ", ".join( zip(*DEFAULT_TARGET_ENCODING_PER_FONT)[0] ))
```
Comment (lephilousophe) [inline devtools/create_bladerunner/subtitles/mixResourceCreator/packBladeRunnerMIXFromPCTLKXLS04.py:250]: You are missing the same kind of fix here.
One small optimization: you can replace `list(zip(*...))[0]` by `next(zip(*...))`.
This avoids building the full list to take only the first element.

Comment (phcoder) [reply devtools/create_bladerunner/subtitles/mixResourceCreator/packBladeRunnerMIXFromPCTLKXLS04.py]: Done. I agree that we should drop python2 support but @antoniou79 is the author and maintainter of this code so I think it's his decisition

**Comment by bluegr** [comment]
@phcoder There's something wrong with the latest push to this branch. It seems like you've pushed the wrong local branch, as all the changes you've done on Python files are gone, and have been replaced by other commits which contain changes unrelated to Python

**Comment by phcoder** [comment]
> @phcoder There's something wrong with the latest push to this branch. It seems like you've pushed the wrong local branch, as all the changes you've done on Python files are gone, and have been replaced by other commits which contain changes unrelated to Python - they are the changes you made in PR #5024

Fixed, thanks

**Review by antoniou79 — APPROVED** [review]
The changes are a good start. Thank you for taking the time to implement this. 
In the past week I've tested the code and made some fixes or changes where needed. 

I'm merging this and a bit later I will apply my changes on top of it.

**Pull request merged**